2022-08-17 00:03:01,355 INFO    MainThread:60427 [internal.py:wandb_internal():90] W&B internal server running at pid: 60427, started at: 2022-08-17 00:03:01.352379
2022-08-17 00:03:01,393 INFO    WriterThread:60427 [datastore.py:open_for_write():75] open: /fsx/ganayu/code/SuperShaper/wandb/offline-run-20220817_000258-338nivuh/run-338nivuh.wandb
2022-08-17 00:03:01,393 DEBUG   HandlerThread:60427 [handler.py:handle_request():141] handle_request: run_start
2022-08-17 00:03:01,428 DEBUG   HandlerThread:60427 [meta.py:__init__():35] meta init
2022-08-17 00:03:01,428 DEBUG   HandlerThread:60427 [meta.py:__init__():49] meta init done
2022-08-17 00:03:01,428 DEBUG   HandlerThread:60427 [meta.py:probe():209] probe
2022-08-17 00:03:01,449 DEBUG   HandlerThread:60427 [meta.py:_setup_git():199] setup git
2022-08-17 00:03:01,569 DEBUG   HandlerThread:60427 [meta.py:_setup_git():206] setup git done
2022-08-17 00:03:01,570 DEBUG   HandlerThread:60427 [meta.py:_save_pip():53] save pip
2022-08-17 00:03:01,573 DEBUG   HandlerThread:60427 [meta.py:_save_pip():67] save pip done
2022-08-17 00:03:01,574 DEBUG   HandlerThread:60427 [meta.py:_save_conda():74] save conda
2022-08-17 00:03:34,208 DEBUG   HandlerThread:60427 [meta.py:_save_conda():84] save conda done
2022-08-17 00:03:34,208 DEBUG   HandlerThread:60427 [meta.py:probe():247] probe done
2022-08-17 00:05:37,025 WARNING MainThread:60427 [internal.py:wandb_internal():153] Internal process interrupt: 1
2022-08-17 00:05:37,336 WARNING MainThread:60427 [internal.py:wandb_internal():153] Internal process interrupt: 2
2022-08-17 00:05:37,336 ERROR   MainThread:60427 [internal.py:wandb_internal():156] Internal process interrupted.
2022-08-17 00:05:37,820 INFO    WriterThread:60427 [datastore.py:close():279] close: /fsx/ganayu/code/SuperShaper/wandb/offline-run-20220817_000258-338nivuh/run-338nivuh.wandb
2022-08-17 00:05:37,824 INFO    HandlerThread:60427 [handler.py:finish():806] shutting down handler
2022-08-17 00:05:38,101 INFO    SenderThread:60427 [sender.py:finish():1111] shutting down sender
2022-08-17 00:05:38,102 INFO    MainThread:60427 [internal.py:handle_exit():80] Internal process exited
